release/servicing/ubuntu14.04/meta.json

```diff
@@ -1,6 +1,6 @@
 {
     "IsLinux" : true,
```

Comment:
All the servicing images can be deleted as the former servicing release is out of support.

Reply:
Sure, I’ll do it when I get back to my pc

Reply:
ALL the servicing images?